logos/benchmarks/benchmark_logos.py

```
@@ -3191,259 +3397,87 @@ def _stop_ollama_docker_via_ssh(
-                        data = json.loads(line)
-                        status = data.get("status", "")
-                        if status:
-                            print(f"  [ollama]   {ollama_name}: {status}        ", end="\r")
-                    except Exception:
-                        pass
-            print(f"\n  [ollama] '{ollama_name}': import complete.")
-        except Exception as exc:
-            print(
-                f"\n  [ollama] WARNING: disk-import of '{ollama_name}' failed: {exc} — will try pull.",
-                file=sys.stderr,
+            result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)